common_components/linux_compat/freertos/freertos_linux.c

```diff
@@ -145,6 +157,13 @@ QueueHandle_t xSemaphoreCreateRecursiveMutex(void)
 
 void vTaskDelete(TaskHandle_t *task)
 {
+    for (int i = 0; i < s_threads; ++i) {
+        if (task == s_notifiers->id) {
```

Comment:
Isn't here an issue regarding the indexes? Seems `s_notifiers` array is being accessed without considering the index?

Reply:
Good catch, thanks! Fixed and updated.